fastlane/Fastfile

```diff
@@ -361,38 +361,45 @@ REPOSITORY_NAME = 'WordPress-Android'
     android_finalize_prechecks(skip_confirm: options[:skip_confirm])
     configure_apply(force: is_ci)
 
+    check_translations_coverage()
+    download_translations()
+
+    android_bump_version_final_release()
+    version = android_get_release_version()
+    download_metadata_strings(version: version['name'], build_number: version['code'])
+
+    # Wrap up
+    removebranchprotection(repository: GHHELPER_REPO, branch: "release/#{version['name']}")
+    setfrozentag(repository: GHHELPER_REPO, milestone: version['name'], freeze: false)
+    create_new_milestone(repository: GHHELPER_REPO)
+    close_milestone(repository: GHHELPER_REPO, milestone: version['name'])
+
+    # Trigger release build
+    trigger_release_build(branch_to_build: "release/#{version['name']}")
```

Comment:
Small note that this is a bit of a misleading `diff` as those lines are just moved to [here below](https://github.com/wordpress-mobile/WordPress-Android/pull/15839/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688L382-L395) without any change.

I'd have expected the diff algorithm to highlight the opposite, i.e. that the `check_translation_progress(…)` calls were the ones deleted here and re-added/moved to the new lane in a couple of lines below instead (which is what happened in practice), but I guess it depends on your point of view and how the diff algorithm favors one interpretation of the change over the other 🤷